### ISSUE TITLE
[Feature Request] Karta zoom/size

### ISSUE THREAD

**Issue by ValooFX** [steffendx/GoNorth#23]
Using any resolution images in karta seems to result in a similiarly small tile that, when loaded initially is pretty tiny (fully zoomed out) and somewhat unreadable when filled with markers.
As it seems to stay approximately the same size (2048 is almost the same as 128), maybe initially load it with a zoom that makes it fit the frame or something similar.

**Comment by steffendx** [comment]
Good idea. I will change the default zoom to fit more in the next release.
